sdk/servicebus/Azure.Messaging.ServiceBus/tests/RuleManager/RuleManagerLiveTests.cs

```diff
@@ -0,0 +1,588 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus.Filters;
+using Newtonsoft.Json;
```

Comment:
Even though this is just for tests, I think we are using system.text.json elsewhere in Track 2, so we should probably be consistent here.